.github/workflows/hrm-lambda-release.yml

```diff
@@ -0,0 +1,113 @@
+name: 'Deploy HRM Lambda'
+on:
+  workflow_dispatch:
+    inputs:
+      lambda_name:
+        description: 'hrm lambda name'
+        required: true
+        default: retrieve-recording-url
+        type: choice
+        options:
+          - retrieve-recording-url
+          - retrieve-transcript
+
+      environment:
+        description: HRM environment to deploy. E.G = development, staging, production (must match with the AWS environment value). Default value = development
+        default: development
+        required: true
+        type: choice
+        options:
+          - development
+          - staging
+          - production
+
+      send-slack-message:
+        description: 'Specifies if should send a Slack message at the end of successful run. Defaults to true'
+        required: true
+        type: boolean
+        default: true
```

Comment:
I think in `workflow_dispatch` we always want to send a Slack message, and maybe not to do so on `workflow_call` event (reusable workflows)?

Reply:
Ahh! I was confused about the workflow call. I'll updated this. Thanks.